src/test/java/com/nhantran/chapter10/LoginTest.java

```diff
@@ -1,63 +1,58 @@
 package com.nhantran.chapter10;
 
 import com.nhantran.base.TestBase;
-import com.nhantran.pages.*;
-import com.nhantran.utils.SeleniumActions;
+import com.nhantran.enums.RailwayTabs;
+import com.nhantran.models.User;
+import com.nhantran.pages.HomePage;
+import com.nhantran.pages.LoginPage;
+import com.nhantran.utils.Constants;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 public class LoginTest extends TestBase {
 
-    HomePage homePage = new HomePage();
-    LoginPage loginPage = new LoginPage();
-
-    public LoginTest() {
-        super();
-    }
-
-    private String loginContentErrorMessage = "There was a problem with your login and/or errors exist in your form.";
-    private String loginAccountErrorMessage = "Invalid username or password. Please try again.";
+    private HomePage homePage = new HomePage();
+    private LoginPage loginPage = new LoginPage();
+    private User validUser = new User(Constants.VALID_USERNAME, Constants.VALID_PASSWORD);
+    private User invalidUser = new User(Constants.VALID_USERNAME, Constants.INVALID_PASSWORD);
+    private User nonActiveUser = new User(Constants.NON_ACTIVE_USERNAME, Constants.NON_ACTIVE_PASSWORD);
 
     @Test(description = "User can log into Railway with valid username and password")
     public void TC001_SuccessToLoginWithValidAccount() {
```

Comment:
the method's name should start with verb (e/x: LoginWithValidAccount, AbleToLoginWithValidAccount,....)